docs/cli-reference.adoc

```diff
@@ -1,4 +1,5 @@
-= Overview of the `odo` CLI Structure
+= Overview of the Odo (OpenShift Do) CLI Structure
```

Comment:
agreed style to for `odo` is to use it in monospace like `odo` if possible or always in lowercase (except when it is at the beginning of the sentence)